### PR TITLE
Test/billsharer

### PR DESCRIPTION
**Pull request by Defibrilator**
Created Balance activity tests and fixed the ones for the Expense activity.

### CODE REVIEW

File: app/src/androidTest/java/com/github/houseorganizer/houseorganizer/BalanceActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    }
+
+    private void openBalances() {
+        onView(withId(R.id.expense_balances)).perform(click());
+        onView(isRoot()).perform(waitFor(500));
+    }
+
+    private void openExpenses() {
+        onView(withId(R.id.balance_expenses)).perform(click());
+        onView(isRoot()).perform(waitFor(500));
```
Comment (Geno22) [inline app/src/androidTest/java/com/github/houseorganizer/houseorganizer/BalanceActivityTest.java:96]: Seems like a nicer way I agree

**Review by Geno22 — APPROVED** [review]
Thanks !
The new waiting way is very elegant great work !

File: app/src/androidTest/java/com/github/houseorganizer/houseorganizer/BalanceActivityTest.java (hunk 1 above)
Comment (aindreias) [reply app/src/androidTest/java/com/github/houseorganizer/houseorganizer/BalanceActivityTest.java:96]: +1

**Review by aindreias — APPROVED** [review]
Nothing much to say, nice work! Thanks for the tests

**Pull request merged**